.github/workflows/test_on_push.yaml

```diff
@@ -78,14 +95,14 @@ jobs:
           path: "${GITHUB_WORKSPACE}/bin"
           key: cache-latest-${{ env.LATEST_COMMIT }}
 
-      - name: Setup Tarantool master
+      - name: Setup Tarantool CE (master)
         if: matrix.tarantool-version == 'master' && steps.cache-latest.outputs.cache-hit != 'true'
         run: |
           tt init
           tt install tarantool master
 
-      - name: Add Tarantool master to PATH
-        if: matrix.tarantool-version == 'master'
+      - name: Add tt Tarantool to PATH
```

Comment:
On line 79, we are adding tarantool 3.0 to PATH, why are we adding it another time here?

Reply:
Thank you, I forgot to clean that up. Fixed.